src/routes/io/io.py

```diff
@@ -80,3 +79,45 @@ async def analyze_img_file(
             "deduced-mime-type": deduced_type})
     except:
         raise internal_error_500()
+
+file_data = {}
+  
+@io_router.post('/user')
+async def create_user(username:str
+,    email:str
+,    password:str
+):
+    try:
+        file_data[username] = {
+            "username" : username,
+            "email" : email,
+            "password": password
+        }
+        return ok_200({"username":username,
+                       "email" : email,
+                       "password": password})
+    except:
+        raise internal_error_500()
+
+@io_router.put('/user')
+async def create_user(old_username:str
+,    old_email:str
+,    old_password:str, new_username:str
```

Comment:
What is the use of old_email & old_password if it is not being used anywhere

Reply:
old email means email which is first created by post request and if wants to change then he can change it with new email